challenge/src/components/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+  
+  }
+
+  render() {
+    return (
+      <button onClick={this.handleClick}>
+        {'Help!'}

```

Comment:
similar idea here, we could potentially pass in the button text through props to make this component more customizeable